modules/distr-tracing-tempo-config-query-frontend.adoc

```diff
@@ -8,60 +8,139 @@
 
 Query is a service that retrieves traces from storage and hosts the user interface to display them.
 
-.Parameters used by the {TempoOperator} to define Query
+.Configuration parameters for the Querier
```

Comment:
It is a little confusing as to what the querier refers to. Specifying throughout could be helpful.

Reply:
Thank you. I'm following up with the SME.

Reply:
Thank you. I've checked with the SMEs and am rewriting this.